Server/SQL/schema.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE IF NOT EXISTS Card
+(
+    id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    deck_id VARCHAR(255) NOT NULL,
+    needs_review BOOLEAN NOT NULL DEFAULT FALSE,
+    front VARCHAR(255) NOT NULL,
+    back VARCHAR(255) NOT NULL,
+    position INTEGER NOT NULL
+
+    FOREIGN KEY (deck_id) REFERENCES Deck(uuid)
+);
+
+
+CREATE TABLE IF NOT EXISTS Deck
+(
+    uuid VARCHAR(255) NOT NULL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    owner VARCHAR(255),  -- theoretically could be null if a user deletes their account?
+    rating FLOAT DEFAULT NULL,
+    num_raitings INTEGER NOT NULL DEFAULT 0,
+    public BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
I think we could have a fourth option where public is false and share_code is a string..